ui/packages/catalog-ui-search/src/main/webapp/component/user-settings/user-settings.view.js

```diff
@@ -18,7 +18,7 @@ var template = require('./user-settings.hbs');
 var CustomElements = require('js/CustomElements');
 var ThemeSettings = require('component/theme-settings/theme-settings.view');
 var AlertSettings = require('component/alert-settings/alert-settings.view');
-var MapSettings = require('js/view/preferences/PreferencesModal.view');
+var MapSettings = require('component/map-settings/map-settings.view');
```

Comment:
Doing this just replaces the 'Layers' settings in MainSettings>MapSettings - is that intentional or is it supposed to be the Units settings in addition to the Layers settings?